internal/controller/state/graph/graph_test.go

```diff
@@ -1269,22 +1269,22 @@ func TestBuildGraph(t *testing.T) {
 	}
 
 	tests := []struct {
-		store        ClusterState
-		expected     *Graph
-		name         string
-		experimental bool
+		store               ClusterState
+		expected            *Graph
+		name                string
+		experimentalEnabled bool
 	}{
 		{
-			store:        createStateWithGatewayClass(normalGC),
-			expected:     createExpectedGraphWithGatewayClass(normalGC),
-			experimental: false,
-			name:         "normal case",
+			store:               createStateWithGatewayClass(normalGC),
+			expected:            createExpectedGraphWithGatewayClass(normalGC),
+			experimentalEnabled: false,
+			name:                "normal case",
 		},
 		{
-			store:        createStateWithGatewayClass(differentControllerGC),
-			expected:     &Graph{},
-			experimental: true,
-			name:         "gatewayclass belongs to a different controller",
+			store:               createStateWithGatewayClass(differentControllerGC),
+			expected:            &Graph{},
+			experimentalEnabled: true,
```

Comment:
It would be nice if we could actually verify that this does something, instead of purely trying to get code coverage